lib/v2/95mm/utils.js

```diff
@@ -0,0 +1,55 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const rootUrl = 'https://www.95mm.org';
+
+module.exports = {
+    rootUrl,
+    ProcessItems: async (ctx, title, currentUrl) => {
+        const response = await got({
+            method: 'get',
+            url: currentUrl,
+            header: {
+                Referer: rootUrl,
+            },
+        });
+
+        const $ = cheerio.load(response.data);
+
+        let items = $('div.list-body')
+            .toArray()
+            .map((item) => {
+                item = $(item);
+
+                const a = item.find('a');
+
+                return {
+                    title: a.text(),
+                    link: a.attr('href'),
+                };
+            });
+
+        items = await Promise.all(
+            items.map((item) =>
+                ctx.cache.tryGet(item.link, async () => {
+                    const detailResponse = await got({
+                        method: 'get',
+                        url: item.link,
+                    });
+
+                    const images = detailResponse.data.match(/src": '(.*?)',"width/g);
+
+                    item.description = `<img src="${images.map((i) => i.split("'")[1]).join('"><img src="')}">`;
```

Comment:
Please use art-template for HTML https://github.com/DIYgod/RSSHub/discussions/8359